uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/view.jsp

```diff
@@ -312,10 +310,10 @@
                                     <c:choose>
                                         <c:when test="${empty featuredPortlet.getParameter('mobileIconUrl')}">
 
-                                            <img src="${defaultIcon}">
+                                            <img src="${defaultIcon}" alt="">
```

Comment:
`alt` with a description can have value when there is a network error.
However even so `alt` with `aria-hidden` is uncommon from my experience.